File: frontend/src/model/Article.ts
export class Article {

  source: string;
  url: string;
  title: string;
  date: string;

  constructor(source: string, url: string, title: string, date: string) {
    this.source = source;
    this.url = url;
    this.title = title;
    this.date = date;
  }
}

File: backend/src/main/java/com/dimka/currencyanalyzer/model/Article.java
package com.dimka.currencyanalyzer.model;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.Instant;
import java.util.Date;

@Accessors(chain = true)
@Data
public class Article {

    private ArticleSource source;
    private String title;
    private String url;
    private Instant date;
}
